uco-master/uco.ttl

```diff
@@ -41,3 +41,27 @@
 		;
 	.
 
+<http://unifiedcyberontology.org/observable#CyberAction>
+	rdfs:subClassOf <http://unifiedcyberontology.org/action#Action> ;
```

Comment:
This should be added directly to the CyberAction definition in observable.ttl not in master.ttl.
I have a separate commit to address this fix.

Reply:
I understand your comment here. To date there has been no ontology import convention. Many direct IRI references have been made throughout these ontology files.
The only import statements recently made were those made when moving to a master.ttl (now uco.ttl). This was the only ontolgy file with both Action and Observable visible via import statements.

If you are saying that the import statements should be embedded in each relevant ontolgy file, I agree.

Reply:
Yes, you are correct that having CyberAction subclassing defined in observable.ttl where it belongs means that an import of Action needs to be added to observable.ttl.
Sorry for the oversight.

Reply:
I am going to import observable into action to gain visibility of CyberAction. I'll make the CyberAction a subclass of Action in action.ttl.  Then I'll import action.ttl into observable.ttl and make CyberAction a subclass of Observable in observable.ttl. I will remove the subclass assertions from uco.ttl.